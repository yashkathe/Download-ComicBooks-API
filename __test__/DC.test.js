const { getDCComics } = require('../index');

test("DC Comics", async () => {
    expect(await getDCComics(1)).toMatchSnapshot();
});