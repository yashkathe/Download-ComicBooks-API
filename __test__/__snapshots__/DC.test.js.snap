// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DC Comics 1`] = `
[
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/Superman-10-Cent-Adventure-1-2003.jpg?fit=400%2C624&ssl=1",
    "description": "It’s the dawn of a new era for the Man of Steel and a great jumping-on point for new readers – and it’s only 10 cents! With the debut of a new Superman creative team, the powerful new Superman villain Amok and the enigmatic Futuresmiths, this is an issue not to be missed – and at this price, you won’t have to! And wait ’til you see the surprise on page 22!",
    "downloadLinks": [
      {
        "Download Now": "https://getcomics.info/links.php/XnJl02KMrU4Hjpj+jefVOG68Jbetny15nO/5LhoGQAGbdRtVSEByKP3v0J4tABz5G6HumLzac3KaW5jTv4RRqQIxw6vnsJ4HG5A61En93+OsuC+QK8p+QOycFFZSMOXB0vHzjhCDKHCZn6lAVXCs8Q==:PtehPVU4QiSxHmTYdTVOoA==",
      },
      {
        "UFILE": "https://getcomics.ufile.io/xn5fs3r5",
      },
      {
        "MEGA": "https://getcomics.info/links.php/ekekaJTwqnC2OTf+/KlUS1HFXkToa+hTjwuztihhu4Un+/fbRXujXrIMZ+s23BrEvOy5gHlIDxCL74j4G/oZmNa/3BufyHVu4dI+odgdJLQ=:UBGyVb9AbW+deGlsgOFvJg==",
      },
      {
        "MEDIAFIRE": "https://getcomics.info/links.php/+jNrH8NVOb6mb3kMT0nyR5L4LJIfEpHuLmo60GBXRfdD2I7VhflAqdqoadl4OAKrKI8A6JR05Ycfg5UWO9EjS7geIn2pLMh6pyySJS7fQ/SkO8Ge62pZGBvt4/MIVYASSj5dV4O0egEJrmRmKEeW1Q==:O0NsM/f+w+jpFt7eRKrFpQ==",
      },
      {
        "ZIPPYSHARE": "https://www14.zippyshare.com/v/lAakpxwu/file.html",
      },
    ],
    "info": {
      "ImageFormat": "JPG",
      "Size": "7 MB",
      "Year": "2003",
    },
    "title": "Superman 10-Cent Adventure #1 (2003)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/Hawkworld-New-Edition-2014.jpg?fit=400%2C615&ssl=1",
    "description": "In this classic 3-issue miniseries from the 1990s, writer/artist Timothy Truman reinvents Hawkman as a brutal member of Thanagar’s police force, in a society split between the decadent ultra-rich and alien immigrants struggling for fortune—and survival! Collects HAWKWORLD #1-3.",
    "downloadLinks": [
      {
        "Download Now": "https://getcomics.info/links.php/uX7uGQAxd4q2Bf4dMdlBapvFmLecMm8yuBBV3Gyq2DDFQBIPQKmLJSzW/ruJrT5fWfeuC+5OZ0BZkeEkkyzQ2Rye0JeFl4ZtMyyO0EAXNWWdDADocsIoLp7Wv9odv6hNHZPJz2cYDM5Z6CBQ6hXUEvvBurF0Yui1kyrGT2x6nuBup4HHklzfIZPLIxlRl3/U:xnlzPTxKyjN2Zqh4fBUwDQ==",
      },
      {
        "UFILE": "https://getcomics.ufile.io/d1ff2tg4",
      },
      {
        "MEGA": "https://getcomics.info/links.php/SAiihQB90JNzAtB37XI370hLM7tcoR/NmWjHDDWuVaVin/VC+d6S/9c+3Vl8fJWukzEzRHqPwrzHhkK+GsmfGylHN0YDoIkHEmIYRR7ZErc=:q5ZwxbMGkZifMq1FFacm1Q==",
      },
      {
        "MEDIAFIRE": "https://getcomics.info/links.php/Vp2Y03ynht90WekVv/amXXZCLH2aA6ppgisDvj1jS1MSUyq0hNRS/h6CKyQawjmK4Jmu1XaSE2epQeH091VwtCKzWsXlewGKB79XwGyhxATnMjxk+gGYoJkw4Kv8Nn3q7sLiCUHMeUkv12mv1AO/TJHrM45G/aWoIxUTcX2aP8AXUhW512KvkMkUyXnjuc/H:TJ/e0rkh0vJOK7v62uwXsg==",
      },
      {
        "ZIPPYSHARE": "https://www50.zippyshare.com/v/raXBrwBL/file.html",
      },
    ],
    "info": {
      "ImageFormat": "JPG",
      "Size": "286 MB",
      "Year": "2014",
    },
    "title": "Hawkworld – New Edition (2014)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/Rogues-4-2022.jpg?fit=400%2C513&ssl=1",
    "description": "Gorilla City blues! All the Rogues wanted was a better life for themselves. But that’s all over thanks to all their greed and backstabbing. Those still left alive are busted up with their backs against the wall, and Grodd’s forces are closing in. With no heroes racing to save them, they must make a deadly deal with Gorilla Grodd to survive.",
    "downloadLinks": [
      {
        "Download Now": "https://getcomics.info/links.php/kYYOTJw5DeDU9EUjXkhq5BrboPM1p9jEZI+V/nVeESDjlTBEgGuKW8uYXSXqZQ4ljd7K3mapdgVprl5smNyqeOjS5LrcYAvvzAfv7VOHs0iVDTZ4nPu6Sf1Bk4Y4O6cOBs241VnansygLEie5yC9IVAa/vk5dVH918x7zpMGFio=:hkT26WJwyA+UgP8XOsBXHA==",
      },
      {
        "UFILE": "https://getcomics.ufile.io/hs9cgey9",
      },
      {
        "MEGA": "https://getcomics.info/links.php/pV5pcDV1s2uQUTg2gCg5XHDw5JwcQ6AoeI5b3U6/5kV9nGOTEA6mG7O6T9OOpzsJCvLJ4m0Y4HEOcSbb69NpAKFqxHePm60LcLEAJVpNamE=:KL6QZIbqm3jBSSfyF7p2Nw==",
      },
      {
        "MEDIAFIRE": "https://getcomics.info/links.php/8E+zDU7GfPHurBX2Bk9tB3JSAshFgtqv0A/fLwYwJ9Ic5+MCK1gNozWThp6ZVQ1L+YFUEuQb+PTw/nGXLPfSnS5DHnqTo3N7bn8rJsdYnEzSAROGR780X39HpHKawRQMAiW2sUQa5bDkCsyLCNWjNdjKh1DRsNGL2C/Hd0VFK9k=:kpT52qTenTvnRnQo6eUGMQ==",
      },
      {
        "ZIPPYSHARE": "https://www61.zippyshare.com/v/rTTLs7h4/file.html",
      },
      {
        "Read Online": "https://readcomicsonline.ru/comic/rogues-2022/4",
      },
    ],
    "info": {
      "ImageFormat": "JPG",
      "Size": "121 MB",
      "Year": "2022",
    },
    "title": "Rogues #4 (2022)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/DC-Horror-Presents-Sgt.-Rock-vs.-The-Army-of-the-Dead-2-2022.jpg?fit=400%2C615&ssl=1",
    "description": "Sgt. Rock and Easy Company are behind enemy lines, armed to the teeth, and ready to go up against the strangest—and deadliest—enemies they’ve ever encountered: zombies, and a whole lot of ’em! Strap in, soldiers, it’s you against the world…of the dead!",
    "downloadLinks": [
      {
        "Download Now": "https://getcomics.info/links.php/zJOcUPONxHDGT3Ic48R8cW7H0PeFbcwyo9DBIiau3LizzfCHM2KCMsiNPak5whnIO6K0xjvbjmOS6ydM4KAWvVcYNASXkGPK500PfWuI114oqpwr01WOCjDFEczXpIQqObyWYuc0h+f2oqsl4JIOF0yAPkeXsw7Zfnol/C7Zhq019uK1Vpu9J+FuqT9o3rf2LBOSThMoJzyeNr5BpjOftKFmK13FUiojVB4IJ0rPDJiQnjarnE0lo8dNRuuvFzIN:xckz+6eoLRk4ipoC+P7xdw==",
      },
      {
        "UFILE": "https://getcomics.ufile.io/q7bi9vyb",
      },
      {
        "MEGA": "https://getcomics.info/links.php/2vH6AsXWQJ9DNzxmcjDgv1rTiSKAhsiICPN28og1Hr9rah4IDznSliW2fwA8BeRFP8q8yutdsIFJuch9LtfvL3hA1L3gW+3Z6/a3F3CTivU=:QbtZ+b2CyzOn+XeuI443Bw==",
      },
      {
        "MEDIAFIRE": "https://getcomics.info/links.php/t5JTq/moTqzdr3heuF3sYKYrvooUTrtOtEZsV/NqghtM7McN6wBFkuR8QVYJYucFMuDpCu99dQpo8sSlw9SS0eUvuHVHFqadKCRyAZ2Y5gJZI3Wnh9r5UoFPqLql8qvj5FqQDrSHGSimK7yBbd7N+ksX2fp0W1220liZ5NDXZ+7rg/JHWfQcAuxP6ZsVnxePyb0eR1/oIQJTlU2A3iMCTfZMSDB0SAS/cPeAYSbv+JY=:ggPI+MVYxsYBKTc4GbTsTw==",
      },
      {
        "ZIPPYSHARE": "https://www61.zippyshare.com/v/8mm1s9GB/file.html",
      },
      {
        "Read Online": "https://readcomicsonline.ru/comic/dc-horror-presents-sgt-rock-vs-the-army-of-the-dead-2022/2",
      },
    ],
    "info": {
      "ImageFormat": "JPG",
      "Size": "31 MB",
      "Year": "2022",
    },
    "title": "DC Horror Presents – Sgt. Rock vs. The Army of the Dead #2 (2022)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/Tim-Drake-Robin-2-2022.jpg?fit=400%2C615&ssl=1",
    "description": "To catch a criminal mastermind, Tim will have to…steal a diamond? Oh no, what’s gotten into you, Tim? It’s Tim Drake like you’ve never seen him! Plus: a murder most foul at the library, the Gotham Marina is in danger, and Bernard isn’t answering Tim’s calls!",
    "downloadLinks": [
      {
        "Download Now": "https://getcomics.info/links.php/fd66uLahIxGN5MQ6xsLkACtPhRgOenT/27sA8TM8u08IU7g5s3OQHjPSYfbfdkt2vRjB5J7AV45GK9sX7yLfZXD/BZyuhiV02vIgARveHJt0lKBP81fktHsBy168+jdegfP+Wl7pa5kiNUAJLwiuT/EsHPDThln69ztdqS520gVkot/D0UZBU/Y7aujOL6to:BylP73EjrdasZzDYpx410w==",
      },
      {
        "UFILE": "https://getcomics.ufile.io/e057jvev",
      },
      {
        "MEGA": "https://getcomics.info/links.php/D5K8Id3Ha1/e56YbDcQ6CLxAD49NGYhy8e3rxodCdaqRoOj6r+zaO1eYPtj05+cjSCyd3m4kmPtVc8twGxOsTe6G+nj095h6fzZLCnUcVVo=:fZVD40dot0lGFrTx1DJJ+Q==",
      },
      {
        "MEDIAFIRE": "https://getcomics.info/links.php/1AEJ3/NVSwUdIA8jA5UE+RWYwNR7l+tGfKOueGwWRDWDCXmYq46yRJeQT+Zk3Mo8bX90Z6DRoTqWnfS8Sq3oLkogoQROilBsesuBFj35FA1OdfmVqOyQIEOWZ7l8hLjXOOU4xHosbXmWUvhkF1uBW7ZkapF08JXKlm/ZEOPpjBGj/AwbyyEkJHqnsF/k+LRM:uBTcrhgS6oJHC9fGunu1ig==",
      },
      {
        "ZIPPYSHARE": "https://www48.zippyshare.com/v/mL7q3K4P/file.html",
      },
      {
        "Read Online": "https://readcomicsonline.ru/comic/tim-drake-robin-2022/2",
      },
    ],
    "info": {
      "ImageFormat": "JPG",
      "Size": "13 MB",
      "Year": "2022",
    },
    "title": "Tim Drake – Robin #2 (2022)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/Punchline-The-Gotham-Game-1-2022.jpg?fit=400%2C615&ssl=1",
    "description": "Punchline is out of jail and back on the streets of Gotham City. She’s not coming back for some of what Gotham has to offer, she’s coming back for all of it. She wants to be the queen of crime, and with the help of her Royal Flush Gang no one will stand in her way. But Punchline created a lot of angry enemies in Gotham before she was incarcerated, and they all want payback. Presenting a miniseries that will redefine the criminal landscape of Gotham City from the superstar creative team of Tini Howard (Catwoman, Knights of X), Blake Howard (Batman: Urban Legends), and Gleb Melnikov (Robin, Wonder Woman)!",
    "downloadLinks": [
      {
        "Download Now": "https://getcomics.info/links.php/MmTsmbelRie/1/ShWJKuJXYV4pwjBqjqVE175DYYpi7kEvuFNeCInaJTMfRLSGllI64qTsRNT4bx+4wd1tSDPGVTYvUOcm5mTZI9+01dgfCBS13l7s8POds2hMjPqvqLZFHjxsb/y62x6phZtSaBYLrELBjotg2uNTHaKs/Z2bB2pt2IYbLWm3mjrm7XxlzS:YB7f7JpNS/GLpZ2e+LG8zQ==",
      },
      {
        "UFILE": "https://getcomics.ufile.io/0z4fdu0p",
      },
      {
        "MEGA": "https://getcomics.info/links.php/wdQCa9ehccbunP2gSW5e7yvM6UCRjLHoTdx7xCoY+UGYMhtpVQL1CPYlNL3DQhHF7TN7+fLQQAfxi0VVb3gSTJjAtRwApQ9htzEpt4vC3Ng=:gqE0LhoT0f3RvjPRdIPjeg==",
      },
      {
        "MEDIAFIRE": "https://getcomics.info/links.php/hfeS1sdvRPGpmqxF+55YYKwhkw2ZaWMjP7BwqHu7GxVk8PFC1iXbOBasuXJulxwca5P1C6PyPCGwHC2exg0KTdfz7KQJMWZ3RpA1sHGk2dYBf/S/wIVIxATnpz5fFkBGWdTmy3MT4gqGMyyq0SIvdj1GHD0XeRDeK63VDNCAt0m0MxnWljyMK6TPSp4HyY0Y:WcQhMiR+FM4XyvWRY2rTjw==",
      },
      {
        "ZIPPYSHARE": "https://www88.zippyshare.com/v/HYfyaFEE/file.html",
      },
      {
        "Read Online": "https://readcomicsonline.ru/comic/punchline-the-gotham-game-2022/1",
      },
    ],
    "info": {
      "ImageFormat": "JPG",
      "Size": "12 MB",
      "Year": "2022",
    },
    "title": "Punchline – The Gotham Game #1 (2022)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/Harley-Quinn-23-2022.jpg?fit=400%2C615&ssl=1",
    "description": "Y’know, bein’ dead is just like ridin’ a bike. There’s a big white light, and then Prince shows up and offers you a chance to host a house party in the clouds with Marilyn Monroe and Abraham Lincoln. Then suddenly you’re brought back to life in a pit of goopy green snot, and you have to find the person who was rude enough to shoot and kill you. To make things even better…turns out that comin’ back to life could have some serious unintended consequences!",
    "downloadLinks": [
      {
        "Download Now": "https://getcomics.info/links.php/NmYDE9+Sr9/ABY6tvD4eSOcvWM7UiinoT7if/bMm9SwEwOCBQthq7Yln54n5ljlDQvc/6z5T2qw1xoZ6vaKz+KVOzy8vvpx4F8PtI6Rpk8EGU49PZD8xuuunWJIXrBwl/pkyvfb85GOM58l2cyWpBNhSoCfenqDr3MMZEQM4rr4=:aQC/S3Kuewb3/a5r80qFiw==",
      },
      {
        "UFILE": "https://getcomics.ufile.io/yev40cjf",
      },
      {
        "MEGA": "https://getcomics.info/links.php/yauG52au0vju3yYLtRNce/eE/RWP0f9LfzFNQ7Q203MTYY8A6H9FRyzgkb+ALKFXOekg6XflkbG0gx2NpUX2hyudG/AD5PUtCXIpzV5JjiI=:r6vV4/FbdnfRQrFfbfhKQg==",
      },
      {
        "MEDIAFIRE": "https://getcomics.info/links.php/OaYlsNrI+wOnpgoTWQjx9Un3DA8S0Yx9KmGd3dqvOwEv1FscwGiL3UZ6GktNHWtMgSgHBTHfDhlq3KizRN3PgBqcoxFCvBu8JFWrR4QvS7m+3wmwiLDS1gSO1DRUG/do93b9KoyiPDLJxof2I4j7qViCqbB7R3H3/JFp/lc+l/Fvftbe3QlqoZrJgUVObMOX:VDOqSiFEMbmkSqE1xRdE5Q==",
      },
      {
        "ZIPPYSHARE": "https://www88.zippyshare.com/v/piwMuRE6/file.html",
      },
      {
        "Read Online": "https://readcomicsonline.ru/comic/harley-quinn-2021/23",
      },
    ],
    "info": {
      "ImageFormat": "JPG",
      "Size": "10 MB",
      "Year": "2022",
    },
    "title": "Harley Quinn #23 (2022)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/Deathstroke-Inc.-14-2022.jpg?fit=400%2C615&ssl=1",
    "description": "Outgunned and overwhelmed, Deathstroke’s first mission takes a shocking and brutal turn as a new threat arises to steal his hit and his reward! Slade has to decide…live or fulfill his contract?",
    "downloadLinks": [
      {
        "Download Now": "https://getcomics.info/links.php/wZsVzl9lK1+gfyI4sz53FffdH8ghogzNZ3sx19KyRGN2EJUBuvtpgTeyEDAlRGMlHK0hnCuUMlnfoqyBNRIZua2v0m4Z0fMjgHvKdaNBJfeLJLnSKJYNjjFbzEWtIz5XgF9M0BLL0L85+2kHtHau2wBeimKrEyFNCyaDCt41jXo=:WRdNTpSDwdOfN5Ehcsdgow==",
      },
      {
        "UFILE": "https://getcomics.ufile.io/cx4fnqec",
      },
      {
        "MEGA": "https://getcomics.info/links.php/dzCURXhb8iLi1FlrJLsVY1Fp/yVDLQSqvyyTk15c58uvdrXE5xuuHijKZat/z4l/ftZjxrwqpK5YnghZSjqxeuDfh9qYY1wivYVLoZzcovA=:9JH1eUvT/h3a+rm7cI6YEw==",
      },
      {
        "MEDIAFIRE": "https://getcomics.info/links.php/QiJCA0DmdAZe9RqnfmbkXMfWV6sgz3JOZkYcVVvTQJy7Y3GgOkIkDc8O8R30xdyWOJ9+dvkJAv7oQtM9+EzZJx28JXDVtSVQ2R7EJ+Cqm3/cTTVsaDb8WxfhqC8nEaie3Ii8Xpw6dhB8j/g9CJFngsyRboCensZq65FhBIVOtSHIzwMOv1ZVxPXDLkvxWNKK:5x9QImll8aTp3b0eOyk92g==",
      },
      {
        "ZIPPYSHARE": "https://www88.zippyshare.com/v/zpvjMZJX/file.html",
      },
      {
        "Read Online": "https://readcomicsonline.ru/comic/deathstroke-inc-2021/14",
      },
    ],
    "info": {
      "ImageFormat": "JPG",
      "Size": "9 MB",
      "Year": "2022",
    },
    "title": "Deathstroke Inc. #14 (2022)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/Catwoman-Lonely-City-4-2022.jpg?fit=400%2C509&ssl=1",
    "description": "ONE OF THE MOST ACCLAIMED BLACK LABEL SERIES EVER REACHES ITS CONCLUSION! All of the sacrifice—all of the broken bodies and shattered relationships—was leading up to this: Selina Kyle standing in front of the Batcave, desperate to know the secrets inside it, secrets once thought to have died with Batman. But you should never open doors you can’t close—if opening them means swinging wide the gates of Hell!",
    "downloadLinks": [
      {
        "Download Now": "https://getcomics.info/links.php/QPfAjM5fwMuHdO1HCD6PnwExstbOs9Qx7EN16Y13TZErtCgqDCoLarYEthjb3fI2cqugyBvTR6dtHv8+SLE3JxfyRwMs85LLd7Hg3h0DVARvR/AU4K0mUBqKzN6YK0JYS4qyZYJLD9HTz11mXn2DwHRqPCDOjEiP11Yiq50LoaY=:40MTuSn00XMggnEm1SkS+Q==",
      },
      {
        "UFILE": "https://getcomics.ufile.io/9x39suhc",
      },
      {
        "MEGA": "https://getcomics.info/links.php/APpIEUDpe/1nGagdho7fH3NGAevrnMaktkmBJQ547aM3V23T/wPa0E616hDmeNaOIGfVFlruRULh1xRwUxRBsvjcVYHJmAtWaIDTdAlmsUo=:xBDd/rpn7K+2FrdYr6YZwQ==",
      },
      {
        "MEDIAFIRE": "https://getcomics.info/links.php/gHJVTf579pemHuLgjcu8TrASa8htXbrJe0TknpVwyGC7Qh+4cLVDDKD5JtsonvDRY86fWGSHo0h3c8Zn76JT5gSM2yU1P/3Aja3LW5SPzPLTuTuguDczEipl5RLLy5ueEpczA2CdLvhVKeFs0M2kqYAE5WrsB2Ei+Tdj2DtuVr0oTCTavqMPHklknYdn60BG:0IL0zpB2R6hOXZs/pTVZmw==",
      },
      {
        "ZIPPYSHARE": "https://www116.zippyshare.com/v/AGFZe98t/file.html",
      },
      {
        "Read Online": "https://readcomicsonline.ru/comic/catwoman-lonely-city-2021/4",
      },
    ],
    "info": {
      "ImageFormat": "JPG",
      "Size": "125 MB",
      "Year": "2022",
    },
    "title": "Catwoman – Lonely City #4 (2022)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/Riddler-Year-One-1-2022.jpg?fit=400%2C615&ssl=1",
    "description": "As depicted in Matt Reeves’s hit movie The Batman, the Riddler wasn’t simply an amusing eccentric with an affinity for wordplay and baffling clues, but as terrifying a villain as any in the annals of the Dark Knight. Here you can see Edward Nashton evolve into the menace known as the Riddler. How did an unknown forensic accountant uncover the dark secrets of Gotham’s underworld and come so close to bringing down the entire city? This six-issue miniseries is an immediate prequel to The Batman—the detailed, disturbing, and at times shocking story of a man with nothing to lose.",
    "downloadLinks": [
      {
        "Download Now": "https://getcomics.info/links.php/2Lg3dt7ptjxITO3yfv/3KH3SIPpTUaf5JM5T0xM+KOjKd8JEnRUbO0BHGl8eK5WH2WLHRKUwOAdvNcekVn+i63LsjeYl3Xay9+I9T6h2OpxX8HVKNWetHUZcTFmLqJ2Ji0ylVp6i3FiAWKUT3rlF8Xr0RdZSnSOFX12hIcilydAQ3D5c23oQBJZhEwtLuIrv:shPsyuqDzlVAyalYWR9Dhw==",
      },
      {
        "UFILE": "https://getcomics.ufile.io/g42uac1x",
      },
      {
        "MEGA": "https://getcomics.info/links.php/fBfIZC0OLjVBIat2Xl7+X5hqehTLYw8CPn6d+EQMRYgAUkllLTcB40413rn0Clbr928+5jb2bWAKh8MTTxcGd3NuCgcPRSR5frjy+G5AEBo=:S65Zfo88dVzN9z/JrsGZag==",
      },
      {
        "MEDIAFIRE": "https://getcomics.info/links.php/a8U6r94GnNaftDlrHo1rqL6kytGX61TRd1iQ1ktLE4LUTFVp2q34MaRBMvog0wXGhElXC1P0Am2ntNPgTr7Z4x/MxIE93Sl9vhsMR4Tqz5iIujaXwmxaxk9aeiOlZPoCYzR3O9yCOXBVu7LXGbR7xiQxeoSlr7CGfV2+yM7XFWzCQrqC76OpVOepSotB+c9Oa3lDmAkgFFFe7xtghV5ilw==:OQnujLfLzWbHfeyWPY4pRg==",
      },
      {
        "ZIPPYSHARE": "https://www88.zippyshare.com/v/Qf4r6j6N/file.html",
      },
      {
        "Read Online": "https://readcomicsonline.ru/comic/riddler-year-one-2022/1",
      },
    ],
    "info": {
      "ImageFormat": "JPG",
      "Size": "57 MB",
      "Year": "2022",
    },
    "title": "Riddler – Year One #1 (2022)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/The-Human-Target-8-2022.jpg?fit=400%2C607&ssl=1",
    "description": "The brutal, Eisner-nominated series continues as time is running out for Christopher Chance to find his murderer. His investigation into the Justice League International takes a violent turn when Rocket Red comes knocking, looking for his friend Guy Gardner!",
    "downloadLinks": [
      {
        "Download Now": "https://getcomics.info/links.php/J3ELh6Cmwz/t4v+AvnsSVlrtKQA8S96ubT4UhQZea7kONi0ZePh89sbjnrT6F+0zyF0/ABtD0U8l+y8MrKKx5Z3SrGz92FRSens7Mw7fUFaJNnDaRa+A8/ycG2wtNEwAtM5NnI9VPoiPFJCGkzBGuLwGCtBtLshbYsnCsSkOCHPlDQrczQjpfnVtVNHxHaz8:mIK+jxGiYQ2IxLdk7YliMQ==",
      },
      {
        "UFILE": "https://getcomics.ufile.io/pv6aje2u",
      },
      {
        "MEGA": "https://getcomics.info/links.php/myMjIJvRgyS97zZNvwi5adb4N0IjD/MzealRnabMPB9NrV7Px4WI0QcbWWmluUCxc03TDz4A9T5Zo/wrd9cbA+ec5AMNDAXXm5A8prwCJ7k=:Cj5KC6eo+fcIR5hCBDrWPw==",
      },
      {
        "MEDIAFIRE": "https://getcomics.info/links.php/Ny0ZOcKTj5gF2JvC9D8beQOGOcin0MDiz5/Ly1Jv4p16tNQYKEkbrXMCXGhHh1z4am4eH578LNSO2Hzx3+03b14XCS1NM6dSlMaENIzcwISSLil6alKM6MvtTGfjlj6z0yuWtRtkZpOl6p4eXz+XIOfonfoeiBELsvBQOZNi6HBmzBYpfJfi2f4z9541RLov:wPQu/OBaexqUU3zXR6IAQw==",
      },
      {
        "ZIPPYSHARE": "https://www88.zippyshare.com/v/cxodJznE/file.html",
      },
      {
        "Read Online": "https://readcomicsonline.ru/comic/the-human-target-2021/8",
      },
    ],
    "info": {
      "ImageFormat": "JPG",
      "Size": "43 MB",
      "Year": "2022",
    },
    "title": "The Human Target #8 (2022)",
  },
]
`;
